compair/api/users.py

```diff
@@ -597,6 +601,132 @@ def get(self):
 
         return {"statuses": statuses}
 
+# /id/lti/users
+class UserLTIListAPI(Resource):
+    @login_required
+    def get(self, user_uuid):
+
+        user = User.get_by_uuid_or_404(user_uuid)
+
+        require(MANAGE, User,
+            title="User's Third Party Logins Unavailable",
+            message="Sorry, your system role does not allow you to view third party logins for this user.")
+
+        lti_users = LTIUser.query \
+                .add_columns(
```

Comment:
you can remove this and instead add
```
lti_consumer_uuid = association_proxy('lti_consumer', 'uuid')
oauth_consumer_key = association_proxy('lti_consumer', 'oauth_consumer_key')
```
to the `lti_user` model

actually after adding the association proxy, you can remove all this and use: 
`lti_users = user.lti_user_links.all()`

Also sort by something